go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/onsi/gomega v1.15.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/stretchr/testify v1.7.0
-	golang.org/x/net v0.1.0
+	golang.org/x/net v0.6.0
```

Comment:
Hi @satyakonduri , will not a Golang update from 1.19 to 1.20 upgrade these dependencies as well. Please check that before proceeding with this PR

Reply:
Will be taking care of these changes as part of [Update Go version to 1.20 ]( https://github.com/dell/dell-csi-operator/pull/121). Hence closing this PR.
Thank You.!